plugins/statistics.koplugin/main.lua

```diff
@@ -36,6 +36,28 @@ local DEFAULT_CALENDAR_NB_BOOK_SPANS = 3
 -- Current DB schema version
 local DB_SCHEMA_VERSION = 20201010
 
+-- This is the query used to compute the total time spent reading distinct pages of the book,
+-- capped at self.page_max_read_sec per distinct page.
+-- c.f., comments in insertDB
+local STATISTICS_SQL_BOOK_CAPPED_TOTALS_QUERY = [[
```

Comment:
May be include the word TIME or DURATION in the name of the variable, because TOTALS what ? :)

Reply:
It includes both time & pages, though ;).

Reply:
> It isn't, it's still counted as (DISTINCT page) ;). Hence why this does the same.

Yes, you are right, sorry. Too many similar names 👍 

But forget me that I'll drill it a little bit more :(

`self.avg_time = (self.book_read_time + self.mem_read_time) / (self.book_read_pages + self.mem_read_pages)`

we are adding book_read_pages with mem_read_pages. And book_read_pages is STATISTICS_SQL_BOOK_CAPPED_TOTALS_QUERY which is count(*) !!